Makefile

```diff
@@ -87,6 +87,11 @@ test-e2e: ## Run e2e tests
 	PULL_POLICY=IfNotPresent $(MAKE) docker-build
 	go test -v -tags=e2e -timeout=1h ./test/e2e/... -args -ginkgo.v --managerImage $(CONTROLLER_IMG)-$(ARCH):$(TAG)
```

Comment:
Do we also need to add a focus here as well? Otherwise wouldn't this make target also cause the conformance tests to run?

Reply:
Oh yes. i missed that. Thanks.